Code/Framework/AzFramework/AzFramework/Physics/Configuration/SceneConfiguration.h

```diff
@@ -42,7 +42,7 @@ namespace AzPhysics
 
         //! Enables reporting of changed Simulated bodies on the OnSceneActiveSimulatedBodiesEvent event.
         //! @note There may be a performance penalty for enabling the Active Actor Notification.
-        bool m_enableActiveActors = false; 
+        bool m_enableActiveActors = true; 
```

Comment:
Is it still accurate the note in the comment for `m_enableActiveActors` saying that there is a performance penalty for using active actors?

Reply:
I think that is referring to this https://docs.nvidia.com/gameworks/content/gameworkslibrary/physx/apireference/files/structPxSceneFlag.html#b4c347372b4433d34d983da780916c53b864a8d54af8c7baaedba117977a523f
which I think means it is a bit more expensive on the PhysX side to use that feature, and they recommend only turning it on if you will actually use it, but I think that inefficiency is massively dwarfed by the inefficiency of all the bus calls for non-moving actors. Probably would be good to reword this comment or just remove it altogether, because it could be pretty confusing.

Reply:
Might be worth making it a settings registry setting, if you think a project that is only/primarily using moving actors would benefit in performance. We can always avoid the extra bus calls by checking against the previous frame's transform if necessary, especially since you're leaving in the functionality for m_enableActiveActors = false instead of removing it completely.

Reply:
yeah, removed the comment since benchmarks showed no difference for tests where every single object moves (i.e. all actors are active).